### ISSUE TITLE
Nothing seems to be working on Fedora 27, Code 1.17.2, Elixir 1.6-dev, otp-20

### ISSUE THREAD

**Issue by hworld** [JakeBecker/vscode-elixir-ls#17]
Hey there, I am trying to use the extension but none of the options seem to be working. Code formatting isn't showing in the menu, there's no hover support, can't go to definition, and no autocomplete.

I use asdf which I see has had issues, but when I manually built/ran the project I get no errors, and when I output ERL_LIBS it seemed correct.

Here's the output log:

```
[Info  - 12:18:52 PM] Child Logger.ErrorHandler of Supervisor Logger.Supervisor started
Pid: #PID<0.83.0>
Start Call: Logger.Watcher.start_link({:error_logger, Logger.ErrorHandler, {true, true, 500}})
Restart: :permanent
Shutdown: 5000
Type: :worker
[Info  - 12:18:52 PM] Application logger started at :nonode@nohost
[Info  - 12:18:52 PM] Application forms started at :nonode@nohost
[Info  - 12:18:52 PM] Application elixir_sense started at :nonode@nohost
[Info  - 12:18:52 PM] Application elixir_ls_utils started at :nonode@nohost
[Info  - 12:18:52 PM] Child ElixirLS.LanguageServer.Server of Supervisor ElixirLS.LanguageServer.Supervisor started
Pid: #PID<0.91.0>
Start Call: ElixirLS.LanguageServer.Server.start_link(ElixirLS.LanguageServer.Server)
Restart: :permanent
Shutdown: 5000
Type: :worker
[Info  - 12:18:52 PM] Child ElixirLS.LanguageServer.JsonRpc of Supervisor ElixirLS.LanguageServer.Supervisor started
Pid: #PID<0.92.0>
Start Call: ElixirLS.LanguageServer.JsonRpc.start_link([name: ElixirLS.LanguageServer.JsonRpc])
Restart: :permanent
Shutdown: 5000
Type: :worker
[Info  - 12:18:52 PM] Application language_server started at :nonode@nohost

```

I don't see any errors and things seem to boot up just fine. Any ideas?

**Comment by JakeBecker** [comment]
I just published v0.2.4 which should make the launching process more robust. Can you give it a try and let me know if it's fixed?

**Comment by hworld** [comment]
Yay, working great now! Thanks so much.
